dvc/command/experiments/init.py

```diff
@@ -97,7 +97,7 @@ def run(self):
 
 def add_parser(experiments_subparsers, parent_parser):
 
-    EXPERIMENTS_INIT_HELP = "Initialize experiments."
+    EXPERIMENTS_INIT_HELP = "Quickly setup any project to use DVC Experiments."
```

Comment:
I don't think we need to mention DVC here. Also, I don't think we need to capitalize "experiments" (at least we don't do in other places).
```suggestion
    EXPERIMENTS_INIT_HELP = "Quickly setup any project to use experiments."
```

Reply:
@jorgeorpinel Do you have an opinion here? Does it matter if we say "experiments" here but "DVC Experiments" in the docs?

I assume the idea is to create a distinction from generic experiments, but since there's no link in the CLI help text, it might make less sense here than in the docs.

Reply:
My 2cs - I agree with Saugat. To be honest even in docs I still have mixed feelings about "DVC Experiments" as a term- if feels a bit artificial.

Reply:
> I don't think we need to mention DVC here.
> Does it matter if we say "experiments" here but "DVC Experiments" in the docs?
> I still have mixed feelings about "DVC Experiments" as a term

I see the point, since it's DVC. FYI the reason we use the branded term "DVC Experiments" in the docs is to distinguish from experiments you may be conducting in a DVC project without using `dvc exp`. Being inside the specific context of `dvc exp` I suppose it's indeed unnecessary here 🙂 esp. since no one loves the term anyway (but it's the best we have ATM for docs at least).